index.js

```diff
@@ -18,16 +22,14 @@ if(global[index.name] && global[index.name].version === index.version) {
 	//detector.find = detection.find;
 	detector.find = function(vid, pid, callback) {
 		// Suss out the optional parameters
-		if(!pid && !callback) {
+		if (isFunction(vid) && !pid && !callback) {
 			callback = vid;
 			vid = undefined;
-		}
-		else if(!callback) {
+		} else if(isFunction(pid) && !callback) {
```

Comment:
Let's add some tests so we don't regress in the future, [`test/test.js#L50-L74`](https://github.com/MadLittleMods/node-usb-detection/blob/875c525cc55b8c8d161138e5055c6b40a15d0851/test/test.js#L50-L74)

Reply:
Tests are added now, I do not own a teensy, so I added the option to override the pid and vid using environment variables. TEST_VID=0x10c4 TEST_PID=0xea60 npm test

Reply:
:+1:

For my own reference (using a mouse) and running on Windows,
```
set TEST_VID=1133&&set TEST_PID=49742&& npm test
```

Reply:
See for more info on an alternative approach to automatically pick a device you already have, https://github.com/MadLittleMods/node-usb-detection/pull/74/files#r262508180